elasticdl/python/common/k8s_tensorboard_client.py

```diff
@@ -75,7 +85,7 @@ def _get_tensorboard_service(self):
             )
             return None
 
-    def get_tensorboard_url(self, check_interval=5, wait_timeout=120):
+    def _get_tensorboard_url(self, check_interval=5, wait_timeout=120):
```

Comment:
Have we tested internally how much time it takes to get the address? I'm little worried here that the master pod will be blocked here for 120 secs, before it can start the training.

Reply:
Good catch. Fixed.